### ISSUE TITLE
[20230625] Weekly AI ArXiv 만담 시즌2 - 21회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#87]
### News
- 6.25 73주년 순국선열들의 희생에 감사드립니다.
- Conferences
  - EMNLP 2023 deadline: 모두 수고하셨습니다.
  - CVPR 2023: 모두들 의미있는 학회 보내셨나요~

- [사실상 쿠데타, 바그너 반란 우크라전 영향은? 우크라 “이제 시작” [월드뷰]](https://n.news.naver.com/mnews/article/081/0003371486?sid=104)
- 일론 머스크 vs. 마크 저커버그
  - 관련 언론기사: https://www.theverge.com/2023/6/21/23769263/mark-zuckerberg-elon-musk-fight-cage-match-worldstar
  - Maye Musk 엄마의 사랑: https://twitter.com/mayemusk/status/1671998408771379201
- [Meta to pull news from Facebook and Instagram in Canada](https://techcrunch.com/2023/06/23/meta-to-pull-news-from-facebook-and-instagram-in-canada/)
  - 네이버는 언론사에 전재료를 지불하고 있습니당
- [Toyota Research Institute unveils generative AI-powered vehicle design tool.](https://venturebeat.com/ai/toyota-research-institute-unveils-generative-ai-powered-vehicle-design-tool/)
- [AWS invests $100 million to help customers succeed with generative AI](https://venturebeat.com/ai/aws-invests-100-million-to-help-customers-succeed-with-gen-ai/)
- [MosaicML의 MPT-30B 공개](https://huggingface.co/mosaicml/mpt-30b-chat)

### ArXiv
- [AudioPaLM: A Large Language Model That Can Speak and Listen](https://arxiv.org/abs/2306.12925)
- [From Word Models to World Models: Translating from Natural Language to the Probabilistic Language of Thought](https://arxiv.org/abs//2306.12672)

**Comment by veritas9872** [comment]
WIP

**Comment by veritas9872** [comment]
# Research

## Full Parameter Fine-tuning for Large Language Models with Limited Resources

ArXiv: https://arxiv.org/abs/2306.09782
GitHub: https://github.com/OpenLMLab/LOMO

<img width="536" alt="image" src="https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/33523965/f0d2c8ce-6a89-4354-a166-e76cd76460b3">

<img width="536" alt="image" src="https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/33523965/9679a799-f333-40ec-9044-948501356828">

LLM finetuning을 진행하기 위한 (다소) 새로운 방법이 제시되었습니다. 언어 모델의 크기가 너무 커지면서 추가 학습을 진행하기 매우 어렵다는 문제점이 있었으나 일반적으로 모델 학습에서 가장 큰 병목은 Optimizer가 보유한 momentum 등 gradient history로 인한 것이었습니다.

LOMO에서는 단순하게 momentum을 모두 제거하고 단순한 SGD 알고리즘만으로 pre-trained LLM을 finetuning 하는 방법을 적용했을 때 LLaMa 65B 모델을 (비교적) low-resource 환경인 8대의 3090Ti GPU를 single node 환경에서 학습할 수 있었습니다.

성능은 gradient state를 기억하는 방법에 비해 부족할 수 밖에 없지만 QAT 혹은 pruning 등 방법을 적용할 때 활용할 수 있는 연구 방향으로 생각됩니다.

**Comment by Clyde21c** [comment]
### Resarch
DeepMind에서 나온 로보틱스 논문 4개.
(로보틱스 전공은 아니지만 LLM decision making 쪽이라.. )

- [SayTap: Language to Quadrupedal Locomotion](https://saytap.github.io/)
  - 걸음걸이 관련 명령이나 상황을 prompt로 주면, 그에 맞는 속도와 적절한 gait으로 4족 보행 로봇을 움직이는 모델
  - LLM(GPT4) -> 속도, 지면 접촉 패턴
  - RL policy -> Sim-to-real motor control
  - e.g. "Good news! We are going to a picnic!" -> (폴짝 폴짝 뛰는 로봇)
![Screenshot from 2023-06-25 17-58-07](https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/35162035/176a972f-6f93-4217-a1ad-38f3cb22920b)

- [RoboCat: A Self-Improving Foundation Agent for Robotic Manipulation](https://www.deepmind.com/blog/robocat-a-self-improving-robotic-agent?utm_source=twitter&utm_medium=social&utm_campaign=robocat)
  - Gato (Cat의 스페인어) 의 후속 논문으로서, General purpose robot을 목표로하는 연구
  - Goal-conditioned approach
  - Gato 아키텍처를 서로 다른 task들에 대한 서로 다른 로봇팔들의 large training dataset으로 pre-train
  - Self-improving: 새로운 task or 로봇팔에 대한 few demonstration (100개~)으로 fine-tune --> 데이터  추가 생성 후 다시 fine-tune 
![Screenshot from 2023-06-25 18-33-49](https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/35162035/94dc1d3c-8b03-435e-897a-2fe3e9d606bc)

- [Language to Rewards for Robotic Skill Synthesis](https://language-to-reward.github.io/)
  - text prompt에 부합하는 복잡한 motion control을 하기위해 Reward Translator라는 중간 단계를 활용 
  - LLM -> (reward) -> model predictive control
  - Reward Translator는 사용자의 명령을 로봇 도메인으로 설명하는 Motion Descriptor와 이를 다시 reward code로 바꾸어 MJPC (Deepmind MuJoCo MPC) planner가 reward 최적화에 사용할 수 있도록 하는 Reward Coder로 구성
![l2r_overview](https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/35162035/ff55d81c-ba5d-45b0-b9ac-7378f3e9a10e)
![reward_translator](https://github.com/jungwoo-ha/WeeklyArxivTalk/assets/35162035/9e3f8b62-bc8f-4d15-88f5-9a0585be0734)

- Foundation model은 아니지만 같이 나온 : [Agile Catching with Whole-Body MPC and Blackbox Policy Learning](https://sites.google.com/view/agile-catching)

### News
- [Stanford Medicine and Stanford HAI Announce RAISE-Health, a Responsible AI Initiative](https://hai.stanford.edu/news/stanford-medicine-and-stanford-hai-announce-raise-health-responsible-ai-initiative)

**Comment by kimyoungdo0122** [comment]
### ArXiv

- [A PhD Student's Perspective on Research in NLP in the Era of Very Large Language Models](https://arxiv.org/abs/2305.12544)
    -  LLM이 발전하면서 여러 언어 관련 생성 task가 함께 발전했지만, LLM이 발전하면서도 아직 나아가야할 점이 많은 NLP 연구 분야 및 task에 대한 소개
    - 아직까지 LLM에 포함되는 Multilingual 기능은 제한적이기 때문에 앞으로 LLM 학습에 추가할 수 있는 다국어 데이터셋 구축 연구 등등
    - Knowledge-base 분야에는 아직까지 여러 전문 지식과 데이터가 필요하기 때문에 이 분야도 연구할 가치가 있음. 특정 도메인이나 전문가 집단에서 활용하기 위해 필요한 작업이기 때문에 여러 NLP 뿐만 아니라 여러 도메인 연구 분야에서도 필요할 것
    -  그 밖에 계산사회과학, 비언어적 특성, 프롬프트 추론, Efficient NLP 등등 연구 분야 제시

---
### News

-

**Comment by terryum** [comment]
지난 한달간 [TechCrunch](https://techcrunch.com/category/artificial-intelligence/)에서 "눈에 보이는대로" 뽑은 AI 소식들 공유 드립니다.

- [[투자] Hyro 시리즈B $20M (약 260억)](https://techcrunch.com/2023/05/31/hyro-secures-30m-for-its-ai-powered-healthcare-focused-conversational-platform/)
: Hyro는 의료 현장의 conversational AI를 하는 곳으로 주로 루틴한 업무 - 예를 들면 환자의 스케쥴을 잡는다던지, 약을 재처방 한다든지 - 의 자동화 효율을 돕는 곳으로 알려져 있습니다. 

- [[투자] Predibase $12M (약 170억, 누적 370억)](https://techcrunch.com/2023/05/31/low-code-ml-platform-predibase-raises-another-12-2m/)
: Predibase는 Low code로 ML을 만들 수 있는 플랫폼으로 이제까지 250개의 모델이 유저들로부터 만들어졌다고 합니다. LLM을 하다보면 실제로 코딩도, 디버깅도 종종 자연어로 하게 되는데요, No code가 좀 더 말이 되는 세상이 오고있는 것 같습니다. 

- [[투자] EvaluAgent $20M (약 260억) 
](https://techcrunch.com/2023/06/23/evaluagent-raises-20m-to-build-out-software-to-evaluate-call-center-agents/)
: AI콜센터 agent를 만드는 스타트업이 아니고, 이들을 평가하는 스타트업입니다. 현재 만들어지고 있는 AI 콜센터 중 1-2%만 퀄리티 평가를 받는다는 점에서 착안했다고 하네

- [[LLM서비스] Voice generation을 하는 ElevenLabs에서 AI합성을 detection하는 기능을 내놨습니다](https://techcrunch.com/2023/06/20/voice-generating-platform-elevenlabs-raises-19m-launches-detection-tool/)
: ElevenLabs는 올해 초 내놓은 매우 realistic 하면서도 빠른 generation을 가능하게 하는 음성합성 서비스로 화제를 일으켰는데요, 내놓자마자 각종 [어뷰징 사례](https://futurism.com/startup-4chan-voice-cloning-ai)(e.g. 연예인 목소리로 인종차별 발언을 합성한다든지)로 곤혹을 치뤘습니다. 이에 따라 AI합성임을 deteciton 하는 기능을 내놓았다고 하네요. 

- [[잡소식] Mechanical Turk 노동자들이 AI를 쓰고 있다고 하네요](https://techcrunch.com/2023/06/14/mechanical-turk-workers-are-using-ai-to-automate-being-human/)
: 아마추어 번역가들이 구글번역기를 쓸 때부터 이미 예견된 일이 아니었을까요

- [[LLM서비스] InstaCart에서 ChatGPT 통합을 내놓았습니다.](https://techcrunch.com/2023/05/31/instacart-in-app-ai-search-tool-powered-by-chatgpt/) 
: ChatGPT를 통해 상품에 대해 묻거나 또는 개인화 된 추천을 받을 수 있습니다. 슬랙, 옐프, 듀오링고, 디스코드 등 많은 곳이 ChatGPT 통합을 내놓았는데요, LLM은 스타트업을 위한 신규서비스가 아니라 기존 대기업을 위한 Add-on 정도일 수 있지 않을까요?
> With Ask Instacart, users can ask questions like “What kind of side dishes should I serve with lamb chops?” or “What fish is similar to salmon?” You can also ask questions like “What are good sauces for grilling chicken?” or “What are dairy-free snacks for kids?”

- [[LLM서비스] Otter에서 내놓은 미팅 기반 챗봇](https://techcrunch.com/2023/06/21/otter-is-introducing-a-meeting-oriented-ai-chatbot/)
: 자동 회의록 작성을 도와주는 서비스 [Otter](https://otter.ai/)에서 미팅의 내용에 대해 물을 수 있는 챗봇 기능을 추가했다고 합니다. 

- [[LLM서비스] 구글의 새로운 검색 환경 Search Lab](https://blog.google/products/search/generative-ai-search/)
: 검색도 새로운 시도들이 이어지고 있는데요, Generative AI를 기반으로 이미지, 음성, 새로운 쇼핑 경험 등 새로운 시도들을 보여준 새로운 검색환경을 구글에서 실험적으로 내놓았습니다.
[[유튜브 영상 링크]](https://youtu.be/dVsiusLQy5Q)

-[[LLM서비스] 구글의 Vertex AI에서 이젠 생성형 AI를 Low code로 쓸 수 있습니다](https://techcrunch.com/2023/06/07/googles-generative-ai-support-in-vertex-ai-is-now-generally-available/)
: 이제는 개발자들이 AI를 잘 몰라도 갖다쓸 수 있습니다, 
> Based on Google’s models like PaLM 2, Imagen and Codey, Vertex AI offers developers access to the PaLM’s features for generating and classifying text, building ChatGPT-like multi-turn chat experiences and a text embedding API for tasks like semantic search and recommendation engines. 

- [[AI서비스] 유튜브의 자동 더빙 서비스 Aloud](https://techcrunch.com/2023/06/23/youtube-integrates-ai-powered-dubbing-tool/)
: 유튜브가 다국어로 자동 더빙해주는 서비스를 내놓았습니다. 이미 1만개 이상의 비디오가 이 서비스를 통해 70개국의 언어로 더빙되었다고 하네요.  

- [[로봇 연구] 집안일을 유튜브로부터 배우는 로봇](https://techcrunch.com/2023/06/22/robots-learn-to-perform-chores-by-watching-youtube/)
:  이번 CVPR에 나온 "Affordances from Human Videos as a Versatile Representation for Robotics"란 연구입니다. [[CVPR영상]](https://youtu.be/Bik4s57iPsY)
